src/pykeen/models/unimodal/distmult.py

```diff
@@ -146,9 +138,9 @@ def score_hrt(self, hrt_batch: torch.LongTensor) -> torch.FloatTensor:  # noqa:
 
     def score_t(self, hr_batch: torch.LongTensor) -> torch.FloatTensor:  # noqa: D102
         # Get embeddings
-        h = self.entity_embeddings(hr_batch[:, 0]).view(-1, 1, self.embedding_dim)
-        r = self.relation_embeddings(hr_batch[:, 1]).view(-1, 1, self.embedding_dim)
-        t = self.entity_embeddings.weight.view(1, -1, self.embedding_dim)
+        h = self.entity_embeddings(indices=hr_batch[:, 0]).view(-1, 1, self.embedding_dim)
```

Comment:
@mberr  is the keyword argument necessary?

Reply:
No. I generally prefer to use keyword whenever possible since it is more robust against changing the order of parameters.

Reply:
I agree in almost all cases, except when it's single argument functions